tests/functional/test_models.py

```diff
@@ -99,12 +112,6 @@ class Employee(models.Model):
     def test_not_found_file_uuid(self, request):
         request.side_effect = InvalidRequestError(404)
 
-        # Just for calling `value.info` inside `Field.clean` method.
```

Comment:
This was just for this test, now for all. It is not clear what is actually happens.